src/sponsoring.ejs

```diff
@@ -0,0 +1,50 @@
+<style>
+    ul li {
+        list-style-type: disc;
+    }
+</style>
+
+<div class="container">
+    <div class="row">
+        <div class="col s12 m12">
+            <h1 class="header center <%= public._data.baseColor %>-text">Want to become a Sponsor?</h1>
+            <h3 class="header center light">Expose your company to a selected group of JavaScript Technology
+                Enthusiasts</h3>
+        </div>
+    </div>
+</div>
+
+<div class="container">
+    <div class="row flow-text">
+        <div class="col s12 m6">
+            <h5 class="header center">You will get ...</h5>
+            <ul>
+                <li>Get up to <b>5 preregistered participant spots</b> for your representatives</li>
+                <li>We place your <b>logo</b> and <b>link</b> on the JSCraftCamp website</li>
+                <li>Your <b>exhibitor</b> will be visible for the whole time during the event</li>
+                <li>Your <b>company name</b> will be mentioned several times during the conference</li>
+                <li>It's an OpenSpace &mdash; contribute the <b>talks that are important to you</b> to the community
+                </li>
+                <li>Sponsors will be announced via the official JSCC <b>Twitter</b> account</li>
+            </ul>
+        </div>
+        <div class="col s12 m6">
+            <h5 class="header center">Why Sponsor?</h5>
+            <p class="block">
+                The JSCraftCamp is an Open Space conference that attracts diverse people from the JavaScript community
+                who really care about their craft. The crowd is self selected by investing time for their own application
+                (pull request).
```

Comment:
I find the sentence `The crowd is self selected by investing time for their own application`  at little bit confusing.
I guess the gross of what you want to say here is that the people attending the conference are there out of free will and curiosity, and are not forced by a company. And they show this by using their own time to put an application in.

What do you think about this:
"All Participants will be there on their private time and by there own drive and curiosity."